articles/search/search-more-like-this.md

```diff
@@ -16,11 +16,11 @@ ms.date: 10/06/2021
 > [!IMPORTANT] 
 > This feature is in public preview under [Supplemental Terms of Use](https://azure.microsoft.com/support/legal/preview-supplemental-terms/). The [preview REST API](/rest/api/searchservice/index-preview) supports this feature.
 
-`moreLikeThis=[key]` is a query parameter in the [Search Documents API](/rest/api/searchservice/search-documents) that finds documents similar to the document specified by the document key. When a search request is made with `moreLikeThis`, a query is generated with search terms extracted from the given document that describe that document best. The generated query is then used to make the search request. By default, the contents of all searchable fields are considered, minus any restricted fields that you specified using the `searchFields` parameter. The `moreLikeThis` parameter cannot be used with the search parameter, `search=[string]`.
+`moreLikeThis=[key]` is a query parameter in the [Search Documents API](/rest/api/searchservice/search-documents) that finds documents similar to the document specified by the document key. When a search request is made with `moreLikeThis`, a query is generated with search terms extracted from the given document that describe that document best. The generated query is then used to make the search request. The `moreLikeThis` parameter cannot be used with the search parameter, `search=[string]`.
```

Comment:
It looks like you dropped a requirement.  Only fields that are "searchable" are in scope for moreLikeThis queries.

Also, although I agree that the original text is confusing, I think it's trying to say that you can constrain moreLikeThis to the fields listed in "searchFields". So instead of all "searchable" fields, it's just "searchFields" fields.

Reply:
I removed this bc it was redundant and less clear than what is already specified by lines 21-22 below.

Reply:
I see it now.  Makes sense.